### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 9

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#143] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181432](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/87c52fe6-ca70-4b74-abdd-e143ad926fc8)

**Comment by CasuFrost** [comment]
$\textbf{punto 1} )$ Posso dire che due gruppi isomorfi hanno la stessa struttura algebrica. $(\mathbb{Z},+)$ è un gruppo ciclico generato da 1, invece $(\mathbb{Q}\backslash0,\cdot)$ non è ciclico, quindi hanno una struttura algebrica differente, e non possono essere isomorfi. So che non è sufficiente come dimostrazione.
$\textbf{punto 2} )$ Se esistesse un isomorfismo $f:\mathcal{S}_3\rightarrow\mathbb{Z}_6$ , allora l'ordine di un elemento dovrebbe essere identico all'ordine della sua immagine. In $\mathbb{Z}_6$, l'elemento $[1]$, ha ordine 6, in $\mathcal{S}_3$ non esiste alcun elemento di tale ordine, quindi $[1]$ non potrà mai essere mappato in un elemento del suo stesso ordine, allora non può esistere un isomorfismo.

**Reply by interestingmaneuver** [reply]
Per 1, sia $f: \mathbb{Z} \to \mathbb{Q}^*$ un omomorfismo suriettivo. Allora si dovrebbe mandare $1\_\mathbb{Z}$ su un generatore di $\mathbb{Q}^\*$, in quanto $f(x) = f(1+1+ \ldots + 1) \text{ (x volte)\} = f(1) \cdot \ldots \cdot f(1) = f(1)^x$, ma siccome non è ciclico non esiste nessun generatore. Quindi non può esistere un omomorfismo suriettivo, né a maggior ragione un isomorfismo.

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/99c1748d-c45a-463d-bef1-72f44ded489f)
